.infrastructure/st-configMap.yml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-map
+  namespace: mysql
+data:
+  init.sql: |
+    GRANT ALL PRIVILEGES ON app_db.* TO 'user_sql'@'%';
+    USE app_db;
```

Comment:
![image](https://github.com/mate-academy/devops_todolist_kubernetes_task_9_working_with_statefuset/assets/87720583/fc74c5b2-459f-4d37-a625-ad2169ed3baf)
Add Eof(End-of-file), need to add empty line at the end

Reply:
fixed